templates/search.html

```diff
@@ -0,0 +1,80 @@
+{% extends "base.html" %}
+
+{% block meta %}
+    {{ super() }}
+    <meta name="robots" content="noindex" />
+{% endblock %}
+
+{% block title %}
+    <title>Search results{% if query %} for "{{query}}"{% endif %} | Documentation for snaps: Universal Linux packages</title>
+{% endblock %}
+
+{% block body %}
+  <div class="p-strip is-shallow u-no-padding--bottom">
+    <div class="row">
+      <div class="col-12">
+        {% if results and results.entries %}
+          <h1 class="p-heading--two u-no-margin--bottom">We've found these results for  <strong>"{{ query }}"</strong></h1>
+        {% else %}
+          <h1 class="p-heading--two u-no-margin--bottom">We couldn't find anything for your search <strong>"{{ query }}"</strong>.</h1>
+        {% endif %}
+      </div>
+    </div>
+  </div>
+
+  {% if  results and results.entries %}
+      {% for item in results.entries %}
+      <div class="p-strip is-bordered is-shallow">
+        <div class="row">
+          <div class="col-12">
+            <h5><a href="{{ item.link }}" class="title-main">{{ item.htmlTitle | safe}}</a></h5>
+            <p>
+              {{ item.htmlSnippet | safe }}
+            </p>
+            <small><a href="{{ item.link }}">{{ item.htmlFormattedUrl | safe }}</a></small>
+          </div>
+        </div>
+      </div>
+      {% endfor %}
+      <div class="p-strip p-pagination">
+          {% if results.queries.previousPage %}
+              <a
+                class="p-pagination__link--previous"
+                href="/search?q={{ query }}&amp;num={{ num }}&amp;start={{ results.queries.previousPage.0.startIndex }}{% if siteSearch %}&amp;siteSearch={{ siteSearch }}{% endif %}"
```

Comment:
Rather than submit `siteSearch` in the query, I think we should hide this and force the `siteSearch` parameter in the back-end. Unlike ubuntu.com, this search should really only ever need to return results from docs.snapcraft.io. So I don't think we need to accept the `siteSearch` parameters at all.

Reply:
Remove from the template.